hpOneView/connection.py

```diff
@@ -271,8 +271,8 @@ def encode_multipart_formdata(self, fields, files, baseName, verbose=False):
         fin.close()
         return content_type
 
-    def post_multipart_with_response_handling(self, uri, file_path, baseName):
-        resp, body = self.post_multipart(uri, None, file_path, baseName)
+    def post_multipart_with_response_handling(self, uri, file_path, custom_headers, baseName):
```

Comment:
I think Felipe has mentioned earlier that new params should be added in the end so that it doesnot break the code or order. is there any reason that custom_headers is not the last param?